pkg/curatedpackages/packageclient.go

```diff
@@ -116,7 +117,56 @@ func (pc *PackageClient) InstallPackage(ctx context.Context, bp *packagesv1.Bund
 	if err != nil {
 		return err
 	}
-	fmt.Println(&stdOut)
+	fmt.Print(&stdOut)
+	return nil
+}
+
+func (pc *PackageClient) ApplyPackages(ctx context.Context, fileName string, kubeConfig string) error {
+	params := []string{"apply", "-f", fileName, "--kubeconfig", kubeConfig}
```

Comment:
I might be completely wrong... but if you guys are referring to `ApplyKubeSpec` (pkg/executables/kubectl.go:189), then it seems like it could easily be modified to handle a nil cluster. 

I tend to agree that the ApplyPackages method feels out of place. It should be at least calling the existing Apply, and not sidestepping right to ExecuteCommand.

For that matter, it seems a little strange that in the exsting ApplyKubeSpec, the `Kubectl` object needs to know how to reach into a `Cluster` object, when it could just accept a `KubeconfigFile` (string), but that's a refactor for another day.

Reply:
Agreed with @ewollesen, `ApplyKubeSpec` should just take a `kubeconfig` string. To avoid adding that refactor to this task, you could just create a `types.Cluster` object with just the kubeconfig in it.